int64 mid_price
int64[] bids_price
int64[] bids_size
int64[] asks_price
int64[] asks_size

